README.md

```diff
@@ -23,12 +18,18 @@ To get a local copy up and running follow these simple steps:
 2. Press the "Code" button and copy the link.
 3. Clone it using git command `git clone <https://github.com/Bria222/KanbanBoard.github.io>`.
 
+## to test the application
+> npm install --save-dev jest
+> npm test
+
```

Comment:
- [ ] Please structure your readme file following the [Microverse](https://github.com/microverseinc/readme-template)
- [ ] Please deploy your project and add the live demo link to your readme file

Reply:
@CollinsTatang  okay thanks let me do so

Reply:
Ok @Bria222

Reply:
@CollinsTatang  i have done the changes can you please check

Reply:
@Bria222  you still have linters errors, please fix it before you submit for review

Reply:
@CollinsTatang We have checked the linters. It is okay.